README.md

```diff
@@ -378,13 +378,14 @@ Examples:
 #### `report -type=text`
 
 ```console
-Requests      [total, rate]             1200, 120.00
-Duration      [total, attack, wait]     10.094965987s, 9.949883921s, 145.082066ms
-Latencies     [mean, 50, 95, 99, max]   113.172398ms, 108.272568ms, 140.18235ms, 247.771566ms, 264.815246ms
-Bytes In      [total, mean]             3714690, 3095.57
-Bytes Out     [total, mean]             0, 0.00
-Success       [ratio]                   55.42%
-Status Codes  [code:count]              0:535  200:665
+Requests                  [total, rate]             1200, 120.00
+Duration                  [total, attack, wait]     10.094965987s, 9.949883921s, 145.082066ms
+Latencies                 [mean, 50, 95, 99, max]   113.172398ms, 108.272568ms, 140.18235ms, 247.771566ms, 264.815246ms
+Bytes In                  [total, mean]             3714690, 3095.57
+Bytes Out                 [total, mean]             0, 0.00
+Response Body Length      [total, mean]             3714690, 3095.57
```

Comment:
I think the display of this information can be improved. This label is now far wider than all the others. What do you think of this alternative?

```
Requests      [total, rate]             240, 50.21
Duration      [total, attack, wait]     4.780706265s, 4.780108709s, 597.556µs
Latencies     [mean, 50, 95, 99, max]   488.965µs, 503.043µs, 571.16µs, 597.908µs, 606.328µs
Bytes In      [total, avg, rx, rx:avg]  22800, 95.00, 22800, 95.00
Bytes Out     [total, avg, tx, tx:avg]  22800, 95.00, 0, 0.00
Success       [ratio]                   100.00%
Status Codes  [code:count]              200:240
Error Set:
```

This would require documenting the semantics of all fields in `ByteMetrics` and putting the new fields there too.

Reply:
We dont measure the body length for bytes out so I dont think there is a reason to add `tx and tx:avg` for that it should always match the total and mean already.

As for bytes in I can change it to have the additional fields, but you also want `mean` changed to `avg`?  Then it wouldnt match the JSON reporter and the latency field.

Reply:
> We dont measure the body length for bytes out so I dont think there is a reason to add tx and tx:avg for that it should always match the total and mean already.

Good point. We should leave them out then even if we loose the consistency.

> As for bytes in I can change it to have the additional fields, but you also want mean changed to avg? Then it wouldnt match the JSON reporter and the latency field.

I think that's OK. Mean and average have the same meaning in this context. It's a trade-off to optimise the information density and visual alignment in the text report while maintaining backwards compatibility in the JSON report.

Reply:
ok I modified the text report and readme on this change.  I'm not sure if I fully understood what you wanted changed so let me know if you wanted it done in another way